src/ngx_http_lua_shdict.c

```diff
@@ -2623,11 +2623,12 @@ ngx_http_lua_ffi_shdict_get(ngx_shm_zone_t *zone, u_char *key,
 int
 ngx_http_lua_ffi_shdict_incr(ngx_shm_zone_t *zone, u_char *key,
     size_t key_len, double *value, char **err, int has_init, double init,
-    int *forcible)
+    int exptime, int *forcible)
```

Comment:
Should be `init_ttl` to avoid any ambiguities here. Also, we should use `long` type here. `int` is 32-bit usually and can easily get overflown.

Reply:
Sounds like a saner name to me :+1: 

Should we do the same modification to this value's type to the already existing implementations of it? Like `ngx_http_lua_ffi_shdict_set_expire`, `ngx_http_lua_ffi_shdict_store`, etc... In another PR of course.

Reply:
@thibaultcha No, they do not have such ambiguities as the `incr()` method.

Reply:
What I mean is that they do use the `int` type as well (I agree their naming is fine).

Reply:
Initial comment addressed.

Reply:
@thibaultcha Oh, right. Better in a separate PR?

Reply:
@thibaultcha Thanks for the catch!